devel/git-sizer/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            github git-sizer 1.0.0 v
+categories              devel
+platforms               darwin
+supported_archs         noarch
+maintainers             @gerardsoleca openmaintainer
```

Comment:
Your "maintainers" line should look like:

`maintainers             {@gerardsoleca obscuredemail} openmaintainer`